fanuc_m10ia_support/launch/test_m10ia7l.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find fanuc_m10ia_support)/launch/load_m10ia7l.launch" />
+  <param name="use_gui" value="true" />
```

Comment:
Will do.  I was just following the structure of the existing m10ia files.  So you may want to update `test_m10ia.launch` as well.